src/pages/admin/artists.tsx

```diff
@@ -1,38 +1,18 @@
 // Componentsj
 import { Heading } from '@radix-ui/themes'
 import AdminLayout from '~/layouts/AdminLayout'
-import SearchContainer from '~/components/SearchContainer'
-import Loading from '~/components/Loading'
 // Utils
-import { api } from '~/utils/api'
-import { DataType } from '~/types/enums'
+import { ArtistsTable } from '~/components/Tables'
 
 export default function AdminArtists() {
-    const {
-        data: artists,
-        isLoading: artistsLoading,
-        refetch
-    } = api.artist.getAll.useQuery()
-
-    const onEdit = async () => {
-        await refetch()
-    }
-
+//moved data-fetching logic to Table components
```

Comment:
prefer you to add comments like this in pr comments like this one as opposed to inline comments

Reply:
@velveteen-rever bump